src/generators/angular/AngularServicesGenerator.ts

```diff
@@ -108,7 +103,8 @@ export class AngularServicesGenerator {
                             ? `Promise<${x.returnType?.type.type}>`
                             : `Observable<${this.getReturnTypeName(x.returnType, x.returnType?.type.type)}>`,
                     statements: (w) => {
-                        x.kind === MethodKind.Download ? this.createDownloadMethod(w, x) : this.createMethod(w, x);
+                        var methodService = new MethodService(w, x, MAPPERS_NAMESPACE, MODELS_NAMESPACE);
```

Comment:
move to constructor

Reply:
I've added new UriService instead of using MethodService, and send UriService to AngularServicesGenerator constructor